tardis/tests/integration_tests/report.py

```diff
@@ -26,22 +26,20 @@
 """
 import datetime
 import json
-import os
-import pkg_resources
 import time
 
 # For specifying error while exception handling
 from socket import gaierror
 
-from py.xml import html, raw
-from pytest_html import __name__ as pytest_html_path
-from pytest_html.plugin import HTMLReport
 import tardis
 
 try:
+    from pytest_html import __name__ as pytest_html_path
+    from pytest_html.plugin import HTMLReport
     import dokuwiki
     import requests
 except ImportError:
+    pytest_html = None
```

Comment:
@yeganer This was the only location where pytest-html caused an ImportError even with integration tests turned off. With that enclosed in try-except, it won't raise any Exception. Not even NoneType error just because it won't be used anywhere.